solid/lib/Controller/StorageController.php

```diff
@@ -439,7 +439,7 @@ private function respond($response) {
                 $policy->addAllowedFontDomain("*");
                 $policy->addAllowedConnectDomain("*");
                 $policy->allowInlineStyle(true);
-                $policy->allowInlineScript(true);
+                // $policy->allowInlineScript(true); - removed, this function no longer exists in NC28
```

Comment:
Should it be documented (somewhere?) that  `allowedScriptDomains` or `evalScriptAllowed` should now be used to enable the `script-src` policy?